lib/msf/core/post/hardware/automotive/uds.rb

```diff
@@ -6,6 +6,16 @@ module Automotive
 
 module UDS
 
+  def initialize(info = {})
+    super(update_info(
+            info,
+            'Compat' => { 'Meterpreter' => { 'Commands' => %w{
+              automotive_active_bus
+              automotive_send_isotp_and_wait_for_response
+      } } }
+          ))
+  end
```

Comment:
Rubocop can fix this automatically in the future, but if you want to make the diff cleaner in the future we can pre-empt that work:
```
          def initialize(info = {})
            super(
              update_info(
                info,
                'Compat' => {
                  'Meterpreter' => {
                    'Commands' => %w[
                      automotive_active_bus
                      automotive_send_isotp_and_wait_for_response
                    ]
                  }
                }
              )
            )
          end
```